spec/lib/asset_sources_spec.rb

```diff
@@ -50,21 +50,24 @@
   end
 
   before do
-    AssetSources.manifest = nil
     File.open(manifest_file.path, 'w') { |f| f.puts manifest_content }
-    allow(AssetSources).to receive(:manifest_path).and_return(manifest_file.path)
-    allow(I18n).to receive(:available_locales).and_return([:en, :es, :fr])
     allow(I18n).to receive(:locale).and_return(:en)
   end
 
   after do
     manifest_file.unlink
-    AssetSources.manifest = nil
+  end
+
+  let(:asset_sources) do
+    AssetSources.new(
+      manifest_path: manifest_file.path, cache_manifest: true,
+      i18n_locales: [:en, :es, :fr]
+    )
   end
```

Comment:
suggestion to refactor out `cache_manifest` into a `let` so it can be overridden below
```suggestion
      manifest_path: manifest_file.path, cache_manifest: cache_manifest,
      i18n_locales: [:en, :es, :fr]
    )
  end
  let(:cache_manifest) { true }
```